GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -391,6 +391,10 @@ public void unregisterView(View view) {
         mRenderableViewGroup.removeView(view);
     }
 
+    public GVRContext getGVRContext() {
```

Comment:
I see the idea. As we discussed, it is a good idea to protect against scene initialization before onInit, but because context is needed for many other things, such a protection cannot be easily enforced. In the worst case, hiding context from subclass of GVRActivity could only make the subclass redundant -- since it cannot do anything without the context. We are currently only use it to specifying an XML -- which could be done in the manifest file as well.

In Android, context is not hidden from subclass of Activity either. So let's bear with it. I would be great rethink the design and resolve all these issues simultaneous but we could do it later.